third-party/spamassassin/src/test/java/org/apache/james/spamassassin/SpamAssassinExtension.java

```diff
@@ -51,8 +53,9 @@ public SpamAssassinExtension() {
                 .withFileFromClasspath("run.sh", "docker/spamassassin/run.sh")
                 .withFileFromClasspath("spamd.sh", "docker/spamassassin/spamd.sh")
                 .withFileFromClasspath("rule-update.sh", "docker/spamassassin/rule-update.sh")
-                .withFileFromClasspath("bayes_pg.sql", "docker/spamassassin/bayes_pg.sql"));
-        spamAssassinContainer.waitingFor(new SpamAssassinWaitStrategy(spamAssassinContainer));
+                .withFileFromClasspath("bayes_pg.sql", "docker/spamassassin/bayes_pg.sql"))
+            .withStartupTimeout(STARTUP_TIMEOUT);
```

Comment:
broken indent ?

Reply:
No.

withFileFromClasspath applies for the dockerfiles

withStartupTimeout applies for the container

Spot the nice `))` at the end of line 56